sympy/tensor/index_methods.py

```diff
@@ -24,6 +23,7 @@ class IndexConformanceException(Exception):
     pass
 
 
+@deprecated(useinstead='EinsteinSum', issue=9284)
 def _remove_repeated(inds):
```

Comment:
This particular function doesn't require it, but other private functions in the file do at the moment since they have doctests that call ``get_indices`` (which I deprecated). E.g. ``_get_indices_Mul``. I could instead kill their doctests?